build_tools/cli/src/lib.rs

```diff
@@ -474,6 +474,18 @@ impl Processor {
                                 ]));
                             config.use_native_runner = true;
                         }
+                        Tests::StdMockDualMicrosoft => {
+                            config.test_standard_library =
+                                Some(StandardLibraryTestsSelection::whitelist(vec![
+                                    "Microsoft_Tests".to_string(),
+                                ]));
+                            config.use_native_runner = false;
+                            config.extra_engine_runner_args = Some(vec![
+                                "--jvm".to_string(),
+                                "--vm.D=polyglot.enso.classLoading=Standard.Microsoft:guest,hosted"
```

Comment:
- The goal of 20529f8b984856ddf153b418968c6b72134de6a1 change is to execute:
```
sbt:enso> runEngineDistribution --jvm --env ENSO_SQLSERVER_DATABASE=tempdb --run test/Microsoft_Tests --vm.D=polyglot.enso.classLoading=Standard.Microsoft:guest,hosted
```
- ideas how to do that properly in our Rust build system are welcomed!

Reply:
Rust build script is hilariously complicated, I suggest putting all the remaining stuff into `extra_engine_runner_args` as well.